targets/csharp/source/source/PlayFabGameserverSDK.cs

```diff
@@ -0,0 +1,203 @@
+﻿// GSDK Only has the following supported frameworks for now
+#if NETSTANDARD2_0 || NETCOREAPP2_1
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PlayFab
+{
+    public enum GameState
+    {
+        Invalid,
+        Initializing,
+        StandingBy,
+        Active,
+        Terminating,
+        Terminated,
+        Quarentined
+    }
+
+    public enum GameOperation
+    {
+        Invalid,
+        Continue,
+        GetManifest,
```

Comment:
>GetManifest [](start = 8, length = 11)

is this even valid peration anymore along with Operation_Count

Reply:
nope, it's not even used, I'll get rid of them

---
In reply to: [255768262](https://github.com/PlayFab/SDKGenerator/pull/398#discussion_r255768262) [](ancestors = 255768262)